drivers/hmis/app/graphql/mutations/submit_household_assessments.rb

```diff
@@ -84,7 +84,7 @@ def resolve(submissions:, confirmed:, validate_only: false)
 
       # Run form processor on each assessment to create/update related records
       assessments.each do |assessment|
-        assessment.assign_attributes(user_id: hmis_user.user_id)
+        assessment.assign_attributes(user_id: hud_user.user_id)
```

Comment:
Is there a reason we're not updating `updated_by_hud_user` when a user submits household assessment(s) from the "Complete Exit/Entry" tab? I think it's probably uncommon that it's a different user from the user who saved them, but it could occur. Elsewhere we seem to be consistent with where we update the `UserID` column, but not here.

Reply:
Thanks, good point, I added this.